lib/src/upgrader.dart

```diff
@@ -381,6 +382,38 @@ class Upgrader {
       barrierDismissible: canDismissDialog,
       context: context,
       builder: (BuildContext context) {
+        if (Platform.isIOS) {
```

Comment:
@ClareKang Is CupertinoAlertDialog only available on iOS? If not, this is not needed.